### ISSUE TITLE
The genus `Ileibacterium` appears twice in the output (genus mode)

### ISSUE THREAD

**Issue by luispedro** [motu-tool/mOTUs_v2#61]
While trying to debug an [NGLess bug report](https://groups.google.com/g/ngless/c/j4ddHhdPOHg), I noticed that the output of motus (database 2.5) seems to be odd in a very particular way

Example

```bash
$ motus profile -c -t 2 -k genus -g 3 -l 75 -n testing -o test.txt -f $reads1 -r $reads2
...
$ grep -c Ileibacterium test.txt
2
```

This is the only genus for which this happens.

**Comment by AlessioMilanese** [comment]
Hi Luis,

I can reproduce even with just:
```
motus profile -k genus -s db_mOTU/db_mOTU_test/test1_single.fastq -o test.motus
```

I will have a look why this happen.

**Comment by AlessioMilanese** [comment]
The issue is with the database.
For `ref_mOTU_v25_10239`, we have the following genus annotation:
```
1980680 Ileibacterium
```
For `ref_mOTU_v25_05939` we have:
```
1937007 Ileibacterium
```

As you can see, the NCBI tax id is different. Since I find the unique names based on the full name (i.e. tax_id + name), they are two different. When I then print it, I remove the NCBI tax id and the result is that there are two different entries with the same name.

The `1980680` is actually wrong, since it maps to _Alterileibacterium_. My guess is that the only genome in `ref_mOTU_v25_10239`, which is `1870997.SAMEA4519299` was reclassified at certain point, or the genome was wrongly annotated. The annotation that I have for progenomes2 is:
```
1870997.SAMEA4519299	2 Bacteria	1239 Firmicutes	186801 Clostridia	186802 Clostridiales	543314 Clostridiales Family XIII. Incertae Sedis	1980680 Ileibacterium	1870997 Ileibacterium massiliense
```

**Comment by AlessioMilanese** [comment]
In any case, we will have an update of the database in 1/2 months, where we will solve this.

For now, can I ask you to run with the `-p` option?
This should solve the problem.
```
$ motus profile -k genus -s db_mOTU/db_mOTU_test/test1_single.fastq -o test.motus -c
  ...
$ grep Ileibacterium test.motus
Ileibacterium	1937007	4
Ileibacterium	1980680	0
```

**Comment by AlessioMilanese** [comment]
Solved with release 2.6.0
